lib/ex_public_key/ex_rsa_private_key.ex

```diff
@@ -1,4 +1,8 @@
 defmodule ExPublicKey.RSAPrivateKey do
+  defimpl Inspect, for: __MODULE__ do
```

Comment:
I don't think you need the `for: __MODULE__` part when the `defimpl` is inside the `defmodule` context.

Reply:
Seems so, didn't know that.